### ISSUE TITLE
Invalid image when installing istio using istioctl

### ISSUE THREAD

**Issue by tharakadesilva** [istio/istio#19027]
**Bug description**
When install istio using istioctl, the image values of istio components' deployments had the prefix `istio` missing from the image section.

For example the pilot deployment's image was set to: `/pilot:1.4.0`.

**Expected behavior**
Expect the image values to be set with the prefix `istio`. Referring to the previous example, it should have been `istio/pilot:1.4.0` instead.

**Steps to reproduce the bug**

- Install istio using the command: `istioctl manifest apply --set profile=demo`

Manually updating the images brings up the cluster.

**Version (include the output of `istioctl version --remote` and `kubectl version` and `helm version` if you used Helm)**

`istioctl version --remote`
```
client version: 1.4.0
control plane version: 1.4.0
data plane version: 1.4.0 (2 proxies)
```

`kubectl version`
```
Client Version: version.Info{Major:"1", Minor:"16", GitVersion:"v1.16.3", GitCommit:"b3cbbae08ec52a7fc73d334838e18d17e8512749", GitTreeState:"clean", BuildDate:"2019-11-14T04:24:34Z", GoVersion:"go1.12.13", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"13+", GitVersion:"v1.13.11-gke.14", GitCommit:"56d89863d1033f9668ddd6e1c1aea81cd846ef88", GitTreeState:"clean", BuildDate:"2019-11-07T19:12:22Z", GoVersion:"go1.12.11b4", Compiler:"gc", Platform:"linux/amd64"}
```

**How was Istio installed?**
Using the command: `istioctl manifest apply --set profile=demo`

**Environment where bug was observed (cloud vendor, OS, etc)**
GKE

**Comment by howardjohn** [comment]
Can you run `istioctl manifest generate --set profile=demo | grep image:`? I don't see this when I run so want to know if it's isolated to apply or generate is somehow different.

What arch is this? Linux/Mac? If Mac, did you install from Homebrew?

**Comment by howardjohn** [comment]
probably Homebrew https://github.com/Homebrew/homebrew-core/issues/46790

**Comment by howardjohn** [comment]
I'll fix this in 1.4.1, for now I think you can explicitly set `--set hub=docker.io`

**Comment by tharakadesilva** [comment]
> probably Homebrew [Homebrew/homebrew-core#46790](https://github.com/Homebrew/homebrew-core/issues/46790)

@howardjohn yes, I am using Homebrew.

> I'll fix this in 1.4.1, for now I think you can explicitly set `--set hub=docker.io`

Thanks for the work around, I will use it for now. And thanks for the fix.

**Comment by tharakadesilva** [comment]
Since this is tracked in Homebrew, closing the issue.
